<?xml version="1.0"?>
<launch>

 <group ns="gripper_controller">

	<node name="gripper_server"  pkg="wsg_32_driver" type="gripper_server" output="screen">
        <param name="ip" type="string" value="192.168.1.53"/>
        <param name="port" type="int" value="1000"/>
        <param name="gripper_speed" type="double" value="0.5"/>
 	 </node>

  	<node name="gripper_joint_relay"  pkg="topic_tools" type="relay" args="/gripper_controller/joint_states /joint_states"/>

    <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen"
          args="joint_state_controller trajectory_controller"/>
  </group>

</launch>

